scripts/test-docs.sh

```diff
@@ -123,7 +123,7 @@ extract_code_blocks()
 
       # Check after the fact: was this file only a class declaration?  If so, we
       # want to put it instead into the next file.
-      has_class1=`grep '^  class' $output_prefix$output_file_display.cpp | wc -l`;
+      has_class1=`grep '^  class\|^  struct' $output_prefix$output_file_display.cpp | wc -l`;
```

Comment:
Just as a note, this change allows the documentation tester to detect both `struct`s and `class`es as things that shouldn't be put inside of an `int main() { }` block.  Previously it would only detect `class`es.